napari/utils/settings/_manager.py

```diff
@@ -0,0 +1,151 @@
+"""Settings management.
+"""
+
+import os
+from json import dumps
+
+import toml
+from pydantic import ValidationError
+from yaml import safe_dump, safe_load
+
+from napari.utils.settings._defaults import (
+    ApplicationSettings,
+    ConsoleSettings,
+)
+
+
+class SettingsManager:
+    """"""
+
+    def __init__(self, config_path=None):
+        self._config_path = os.path.join(
+            os.path.expanduser("~/"), ".napari", "settings"
+        )
```

Comment:
You can start to use pathlib if you want :-)

Reply:
Oh yes... I am just old school :), will do!